include/constants/metatile_behaviors.h

```diff
@@ -30,9 +30,15 @@
 #define MB_UNUSED_SOOTOPOLIS_DEEP_WATER_2 0x1A
 #define MB_STAIRS_OUTSIDE_ABANDONED_SHIP 0x1B
 #define MB_SHOAL_CAVE_ENTRANCE 0x1C
+#if OW_AUTO_SIGNPOST == TRUE
```

Comment:
Do they have to be preroc'd?

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5044/commits/9ca940719f037b9a132510ceac9d19436653e64a